.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+minimum_pre_commit_version: '2.17'
+repos:
+  # shell scripts
+  - repo: https://github.com/detailyang/pre-commit-shell
+    rev: 1.0.5
+    hooks:
+      - id: shell-lint
+        args: [ --format=json ]
+
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-docstring-first
+      - id: check-merge-conflict
```

Comment:
@TheoBrigitte I'm fine with removing this.

This was added as its part of the checks we have for Python.
https://github.com/giantswarm/github/blob/30cf52fd5ef200889e23c7040e629919b5158ebe/languages/python/.pre-commit-config.yaml

@piontec WDYT?

Reply:
The name is confusing. It doesn't check for merge conflict, but for `<<<<<` markers of a conflict that needs to be resolved still present in the file :) So all it does is a quick text check to prevent you from committing unmerged conflicts.
https://github.com/pre-commit/pre-commit-hooks#check-merge-conflict

Reply:
This is useful but yes the name is confusing. I've added a comment to make it clearer.

I also removed check-docstring-first as it doesn't apply for Go.

Reply:
Ok that's much better then, let's keep it and thanks for the comment.